bench/workgen/workgen.cxx

```diff
@@ -2237,7 +2354,7 @@ int WorkloadRunner::run_all(WT_CONNECTION *conn) {
             _trunners[i]._stop = true;
     if (options->sample_interval_ms > 0)
         monitor._stop = true;
-    if (options->oldest_timestamp_lag > 0 || options->stable_timestamp_lag > 0) {
+    if (options->oldest_timestamp_lag > 0 || options->stable_timestamp_lag > 0 || options->max_idle_table_cycle > 0) {
```

Comment:
This is correct, but I think it would be easier to unconditionally set `stopping = true`.

Reply:
Updated